src/logic/operate.js

```diff
@@ -12,7 +12,6 @@ function operate(numberOne, numberTwo, operation) {
   }
   if (operation === "x") {
     result = num1.times(num2);
-    window.result = result;
   }
   if (operation === "÷") {
     if (numberTwo === 0) return "Error: Division by zero";
```

Comment:
- [ ] [REQUIRED] Please add a test unit for this operation as I got an error related to proptype when I divide number by 0